doc/source/rllib/key-concepts.rst

```diff
@@ -115,40 +116,32 @@ You can `configure the parallelism <rllib-training.html#specifying-resources>`__
 Check out our `scaling guide <rllib-training.html#scaling-guide>`__ for more details here.
 
 
-Policies
```

Comment:
We should leave policies out of the concepts from now on.
They are only used on the sampling stack and therefore transparent.